src/bottomSheet/BottomSheet.js

```diff
@@ -40,6 +47,13 @@ class BottomSheet extends Component {
             containerStyle && containerStyle,
           ])}
         >
+          {touchOutsideToClose && (
+            <TouchableOpacity
```

Comment:
I think here `TouchableWithoutFeedback` would be a better choice.

Reply:
Tried this choice. But `TouchableWithoutFeedback` doesn't allow empty child here.

Reply:
Checkout here how the `Overlay` code is able to do it: https://github.com/react-native-elements/react-native-elements/blob/next/src/overlay/Overlay.js

Reply:
I know how to do it. But I didn't see any benefits in that way.